templates/terraform/examples/base_configs/test_file.go.erb

```diff
@@ -0,0 +1,12 @@
+<%= lines(autogen_notice :go) -%>
+
+package google
+
+import (
+	"fmt"
+)
+
+<% object.example.each do |example| -%>
+func testAcc<%= product -%><%= resource_name %>_<%= example.config_test -%>
```

Comment:
I think i'd prefer if you just use 
```erb
func testAcc<%= product -%><%= resource_name %>_<%= example.name -%>(prefix string) string {
<%= example.config_test -%>
}
```

It makes me very nervous to have a template start in the middle of a line that needs to be unbroken. I haven't looked too closely at the other files to see if they do this but if they do we should change that